app/models/embed/purl.rb

```diff
@@ -46,7 +46,7 @@ def copyright
     end
 
     def license
-      cc_license || odc_licence
+      mods_license || cc_license || odc_license
```

Comment:
I don't think this will do the right thing with the old-style MODS licenses for CC v3 content, e.g.

https://purl.stanford.edu/gw298sq0504.xml

(Not that that particular object has the rights panel, but... presumably there's a real example out there?)

I think it'd be fine to check cc/odc first, or we could use the `mods_display` gem here to just render it?

Reply:
@cbeer good :eye:. Take two is now up. LMK if that's sufficient or if we need the `mods_display` dependency. :man_shrugging: